gtest/src/blocks.rs

```diff
@@ -29,35 +29,42 @@ use crate::BLOCK_DURATION_IN_MSECS;
 thread_local! {
     /// Definition of the storage value storing block info (timestamp and height).
     static BLOCK_INFO_STORAGE: RefCell<Option<BlockInfo>> = const { RefCell::new(None) };
```

Comment:
Consider to use `Rc`. 
```suggestion
    static BLOCK_INFO_STORAGE: Rc<RefCell<Option<BlockInfo>>> = const { Rc::new(RefCell::new(None)) };
```
And
```rust
pub(crate) struct BlocksManager(Rc<RefCell<Option<BlockInfo>>>);
```

So you can use `Rc::strong_count`  in `Drop` implementation.

Reply:
done